TestWebService/Sources/TestWebService/TestWebService.swift

```diff
@@ -80,9 +81,12 @@ struct TestWebService: Apodini.WebService {
             )
         }
         
-        HTTP(rootPath: "http")
+        HTTP(rootPath: "http") {
+            HTTPAuditor()
```

Comment:
I would suggest to name the auditors the same for all exporters. As noted on chat.tum.de: I think it makes sense to even use the same `InternalConfiguration` for the REST and HTTP Exporters.

Reply:
I've merged the two `InternalConfigurations`, which were already the same anyways.

Currently, the type parameter to a `DependentStaticConfiguration` is the outward facing `REST` or `HTTP` configuration, not the internal configuration. Thus, merging the `HTTPAuditor` and `RESTAuditor` is currently not possible, as this would require conforming to the same protocol twice.

If we were to take the internal configuration as a type parameter to `DependentStaticConfiguration` (which was your suggestion anyways @PSchmiedmayer 😅), then that would force us to make the `OpenAPI` configuration available as a dependent of the `HTTP` as well as the `REST` configuration. That's fine by me, and @PSchmiedmayer suggested it too.

Do we need to make any modifications to the OpenAPI exporter for it to work as a dependent of the HTTP exporter? Who would I best talk to for this @PSchmiedmayer?

Reply:
Thanks for the response. I took a look at the diff and the current state and think that making the `OpenAPI` configuration available as a dependent configuration of the `HTTP` and the `REST` Exporter is definitely a good idea and would not require too many changes to the Exporter as far as I can see. 

I see two main differences that would need to be addressed:
1. The response type of an endpoint: The REST Exporter wraps a response in the `ResponseContainer` to add the HATEOAS information to the responses, which the HTTP Exporter does not do. Therefore it would make sense to add the way a response is packaged at the end to the `HTTPExporterConfiguration`. The REST Exporter would share with the OpenAPI Exporter that it uses the `ResponseContainer`, and the HTTP Exporter would share that it doesn't use a wrapping. Moving the `ResponseContainer` type to the ApodiniHTTPProtocol target would then probably be a reasonable idea.
2. The HTTP Exporter should Exporter all request-response Handlers the same way as the REST Exporter does. The exporters currently use different mechanisms to register the endpoints but should result in the same final product. That this is the same could only be guaranteed if they would share the code for registering the routes, e.g., the ApodiniHTTPProtocol target. The only area where this registration differs is in the streaming cases for Handlers (e.g., client-side stream). I am not sure if and how OpenAPI could reflect HTTP/2 streaming behavior, but this might then require some additional code in the OpenAPI Exporter. But this feature is currently still on the roadmap and not implemented, so it would not require immediate changes in the Exporter at the moment, and when you tackle this, we can see how and if this can be incorporated into the OpenAPI Exporter 👍

Reply:
That sounds good! Point 1 makes a lot of sense.

Would you include moving the route registration to `ApodiniHTTPProtocol` in this pull request? It seems like that is a bigger task which might deserve its own PR to me.

Reply:
I implemented the correct schema exporting for HTTP with OpenAPI! It looked to me like the modifications I made were the only ones necessary, right?